src/backend/activity.nim

```diff
@@ -30,7 +30,7 @@ type
 
   # see status-go/services/wallet/activity/filter.go Status
   ActivityStatus* {.pure.} = enum
-    Failed, Pending, Complete, Finalized
```

Comment:
Just a note on the original intention here on this generic interface for activity entries. For multi-transaction entries that might have multiple `Confirmed`, or not, sub-transactions, `Complete` was showing when all sub-transactions got `Confirmed` status. In the case of regular transactions `Complete` = `Confirmed`.